apollo-runtime/src/main/java/com/apollographql/apollo/cache/ApolloCacheHeaders.java

```diff
@@ -0,0 +1,21 @@
+package com.apollographql.apollo.cache;
+
+import com.apollographql.apollo.cache.normalized.NormalizedCache;
+
+/**
+ * A collection of cache headers that Apollo's implementations of {@link NormalizedCache} respect.
+ */
+public final class ApolloCacheHeaders {
```

Comment:
According to our checkstyle, interfaces should have "behavior" and hence methods. I would prefer if this was an interfrace, but would have to change our checkstyle.